src/data/daily_providers_metrics.csv.sh

```diff
@@ -16,8 +16,8 @@ COPY (
 
     select
         date,
-        count(distinct provider_id)  filter (where spark_retrieval_success_rate > 0.90) as meet_retrieval_sli,
-        count(distinct provider_id)  filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_sli,
+        count(distinct provider_id) filter (where spark_retrieval_success_rate > 0.90) as meet_retrieval_sli,
+        count(distinct provider_id) filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_sli,
```

Comment:
```suggestion
        count(distinct provider_id) filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_slo,
```

Is there a reason this isn't `>= 0.95`?  Is it possible to have a sector_health_rate above 1?

Reply:
Shouldn't be possible. We can clean that up.